orchagent/acltable.h

```diff
@@ -59,3 +61,28 @@ typedef enum
 } acl_table_type_t;
 
 typedef std::unordered_map<std::string, acl_table_type_t> acl_table_type_lookup_t;
+typedef std::map<std::string, sai_acl_stage_t> acl_stage_lookup_t;
+typedef std::map<sai_acl_stage_t, sai_switch_attr_t> acl_stage_to_switch_attr_lookup_t;
+
+struct AclTableGroupMember {
```

Comment:
Please move to {-> new line and for other structs

Reply:
Fixed